### PR TITLE
Geoportti updates

### PR DESCRIPTION
**Pull request by samumantha**
These are now based on what I took away from email, no problem to update more, if needed:

* Rename course to Geocomputing on the supercomputer
* Replace upper left image
* Remove Copyright notice from course page
* Remove Copyright holder from license file
* Remove author from course page
* Rename CSC section to Background info
* Rename CSC page to Organizers
* Replace main banner image -> all organizer logos instead of only CSC (not EuroCC)
* Replace favicon (was CSC logo) with same as upper left image
* update to CC BY-SA license (in license file, remove legal text)
* add organizer logos to html image (upper left, always visible); multiple options provided

Looks like this now: 
image

### CODE REVIEW

**Comment by ktiits** [comment]
License should be Creative Commons BY-SA 4.0. And it would be nice to get it to footer of the rendered pages. I would like to avoid the copyright word, so was wondering, if we could do it so that you put back the Authors attribute and we would have there "CSC - IT center for science. This page is available under the Creative Commons 4.0 Share-Alike License."?

File: conf.py [hunk 1]
```diff
@@ -30,10 +30,10 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
 # setting title in left panel (default is <projectname> documentation <release>)
-#html_title = "CSC Geocomputing course"
+#html_title = "Geocomputing course"
 # setting upper left logo
-html_logo = "materials/images/bg.png"
-html_favicon = "materials/images/csc.png"
+html_logo = "materials/images/course_name.png"
+html_favicon = "materials/images/course_name.png"
```
Comment (ktiits) [inline conf.py:36]:
```suggestion
html_favicon = "materials/images/csc.png"
```

File: conf.py [hunk 2]
```diff
@@ -30,10 +30,10 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
 # setting title in left panel (default is <projectname> documentation <release>)
-#html_title = "CSC Geocomputing course"
+#html_title = "Geocomputing course"
```
Comment (ktiits) [inline conf.py:33]:
```suggestion
#html_title = "Geocomputing with Puhti supercomputer course"
```

File: index.rst [hunk 3]
```diff
@@ -3,11 +3,11 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-.. image:: ./materials/images/banner.png
+.. image:: ./materials/images/banner_all_organizers2.png
     :class: dark-light
 
-Welcome to CSC Geocomputing course!
-====================================
+Welcome to Geocomputing on the supercomputer course!
```
Comment (ktiits) [inline index.rst:9]:
```suggestion
Welcome to Geocomputing with Puhti supercomputer course!
```

File: conf.py [hunk 4]
```diff
@@ -7,9 +7,9 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = 'Geocomputing course'
-copyright = '2023, CSC - IT center for Science'
-author = 'CSC - IT center for Science'
-release = '0.1'
+#copyright = '2023, CSC - IT center for Science'
+#author = 'CSC - IT center for Science'
```
Comment (ktiits) [inline conf.py:11]:
```suggestion
author = 'CSC - IT center for Science. This page is available under the Creative Commons 4.0 Share-Alike License.'
```

File: index.rst [hunk 5]
```diff
@@ -34,9 +34,9 @@ Table of contents
 
 .. toctree::
    :maxdepth: 2
-   :caption: CSC - IT Center for Science
+   :caption: Background info
```
Comment (ktiits) [inline index.rst:37]:
```suggestion
   :caption: CSC - IT Center for Science
```

File: index.rst [hunk 6]
```diff
@@ -34,9 +34,9 @@ Table of contents
```
Comment (ktiits) [inline index.rst:34]: materials/orga.md
   materials/examples.md

Comment (ktiits) [inline index.rst:34]:
```suggestion
   materials/orga.md
   materials/examples.md
```

File: index.rst [hunk 7]
```diff
@@ -34,9 +34,9 @@ Table of contents
 
 .. toctree::
    :maxdepth: 2
-   :caption: CSC - IT Center for Science
+   :caption: Background info
 
-   materials/csc.md
+   materials/orga.md
    materials/examples.md
```
Comment (ktiits) [inline index.rst:40]:
```suggestion

```

File: conf.py (hunk 2 above)
Comment (samumantha) [reply conf.py:33]: (this one is not used, it would be used as text if no html image were given.

**Comment by samumantha** [comment]
image
or 
image
or any other variation of these better image?

**Comment by samumantha** [comment]
Personal favorite: 
image

**Pull request merged**